src/org.freedesktop.PackageKit.xml

```diff
@@ -476,6 +476,17 @@
       </doc:doc>
     </signal>
 
+    <!--*********************************************************************-->
+    <signal name="InstalledChanged">
+      <doc:doc>
+        <doc:description>
+          <doc:para>
+            This signal is emitted when the number of installed apps could possibly change
```

Comment:
```suggestion
            This signal is emitted when the set of installed apps has potentially changed.
```

It’s not just about the number of apps, it’s also emitted when the set of apps changes, right? So if, for example, a package were transitioned to another one on upgrade, but the number of installed packages doesn’t change, this signal should still be emitted, yes?

Reply:
In case of the dnf plugin, yes, it's emitted whenever the RPM database changes, no matter why (the count/number/set can be the same, only with different versions, like when doing update/downgrade of a package).